docs/adr/lts-strategy.md

```diff
@@ -0,0 +1,176 @@
+- `express` version 4.x is considered a special case. It will receive a longer `MAINTENENCE` phase of 18 months or until
+  the TC agrees we are save cutting support.
+- `express` dependency packages will follow the same timeline and support dates for the `express` version which they are
+  included with.
+- Middleware packages should follow the timeline and support dates for the `express` versions they are compatible, but
+  should make reasonable and informed decisions which can deviate if necessary.
+- All other packages are suggested to define an appropriate strategy for support.
+
+### Node.js Version Support
+
+- Each major version of all packages (`express`, deps, middleware, etc) will support a minimum of the oldest Node.js LTS
+at the time of release and maintain that support through its entire life cycle. This includes EOL Node.js versions.
+- `express` may choose to continue support for EOL Node.js versions in all three phases if deemed valuable, but will
+strive to cut support before it becomes a drain on progress.
+- All `express` dependency packages must support, at a minimum, the Node.js versions that are supported by the `express`
+major line they are included in while that `express` line is not `EOL`.
+- Middleware packages must at a minimum follow the same Node.js versions as the `CURRENT` and `ACTIVE` release lines of
+`express`. They should attempt to also support `MAINTENANCE` lines as well, but the discretion of the captains is
+important.
+- All other packages are suggested to follow a reasonable Node.js support policy for their needs.
+
+### Life Cycle Phases & Meaning
+
+**CURRENT**
+
+- New majors will go through a short period hardening to ensure stability, security, and ecosystem libraries/resources
+  compatibility.
+- We will strive to ensure no breaking changes are included, but reserve the right to make security or high priority
+  fixes of breaking nature within this period.
+- `CURRENT` lines will receive all types of active work including: bug fixes, security patches, new features, and
+  deprecation notices.
+- Users are recommended to use  `CURRENT` lines and to upgrade as quickly as their risk profile allows
+
+**ACTIVE**
+
+- `ACTIVE` lines will receive all types of active work including: bug fixes, security patches, new features, and
+  deprecation notices.
+- For users, `ACTIVE` lines are considered the most stable and well supported version at any given time.
+
+**MAINTENANCE**
+
+- `MAINTENANCE` lines will only receive security patches or high priority bug fixes.
+- Users are highly encouraged to upgrade to a `CURRENT` or `ACTIVE` release.
+
+### Guidelines for other packages/repos
+
+**`express` dependencies**
+
+### Schedule
+
+For the existing release lines, we will set the following phase dates:
+
+
+| Major | CURRENT | ACTIVE | MAINTENANCE | EOL |
+| ----- | ------- | ------ | ----------- | --- |
+| 4.x   |         |        | 2025-04-01 | 2026-10-01 or later |
```

Comment:
```suggestion
| 4.x   |         |        | 2025-04-01 | 2026-10-01 |
```

We can always "negociate" but let's avoid setting expectation to delay that day forever :)

Reply:
This was at @ctcpip's request: https://openjs-foundation.slack.com/archives/C02QB1731FH/p1743197622883289?thread_ts=1743196832.158829&cid=C02QB1731FH

Reply:
If we are setting a date, we should have reasonable confidence that we would actually meet that date.  Otherwise, it's just completely arbitrary.  I suggest we leave as is, or put `TBD`.

Reply:
Changed to TBD with a bit of extra language in the footnote.